README.yaml

```diff
@@ -69,9 +77,13 @@ usage: |-
   Here's how to invoke this example module in your projects
 
   ```hcl
-  module "example" {
-    source = "https://github.com/cloudposse/terraform-aws-s3-config.git?ref=master"
-    example = "Hello world!"
+  module "aws_config_storage" {
+    source                   = "git::https://github.com/cloudposse/terraform-aws-s3-aws-config.git?ref=master"
```

Comment:
Let's rename to `cloudposse/terraform-aws-config-storage`; I think we should keep `aws-config` as that is what the AWS service is called. Let's use "storage" since we have a "storage" module already. If we can re-use our log-storage module, that's ideal. Note, the current repo name on github does not match this current `source`

Reply:
or `cloudposse/terraform-aws-config-s3-logs` or `terraform-aws-config-s3-log-storage`